en/docs/includes/design/add-oas-example.md

```diff
@@ -0,0 +1,170 @@
+    With OpenAPI 3.0 provide mock response body and headers following the below formats. 
+
+    **Single Example for an Operation**
+
+    ```yaml tab='Format'
+    <resource path>:
+      <operation>:
+        responses:
+          <response code>:
+            description: <description>
+            headers:
+              <header>:
+                example: <header example value>
+            content:
+              <media type>:
+                example: <example>
+    ```
+
+    ```yaml tab='Example'
+    /pet/findByStatus:
+      get:
+        responses:
+          '200':
+            description: OK
+            headers:
+              x-wso2-example:
+                example: example header value
+            content:
+              application/json:
+                example:
+                  mock response: hello world
+    ```
+
+    **Multiple Examples for an Operation**
+
+    ```yaml tab='Format'
+    <resource path>:
+      <operation>:
+        responses:
+          <response code>:
+            description: <description>
+            headers:
+              <header>:
+                example: <header example value>
+            content:
+              <media type>:
+                examples: 
+                  <example reference>
+                    value: <example>
+    ```
+
+    ```yaml tab='Example'
+    /pet/findByStatus:
+      get:
+        responses:
+          50X:
+            description: Service Unavailable
+            headers:
+              x-wso2-example:
+                example: example header value
+            content:
+              application/json:
+                examples:
+                  ref1:
+                    value:
+                      mock response: hello world
+                  ref2:
+                    value:
+                      mock response: Welcome
+          default:
+            description: default response
+            headers:
+              x-wso2-example:
+                example: default header value
+            content:
+              application/json:
+                examples:
+                  ref1:
+                    value:
+                      mock response: default hello world
+                  ref2:
+                    value:
+                      mock response: default Welcome
+    ```
+
+    | **Place Holder** | **Usage** |
+    |-----------------|--------------------|
+    | `response code` | Can be 3 digit status code or a wildcard format like 2XX. `default` can be also provided instead of a particular status code. |   
+    | `header` | Header name. You can provide multiple headers similarly under `headers`. |   
```

Comment:
These are definitions specifically of the placeholders rather than the parameters. Here the placeholder `header` given in the `format` section specifically refers to one header. And Amali is just pointing out that it is possible to add multiple in the same way.